core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -640,11 +640,11 @@ private void freeSpaceInternal(long sessionId, long availableBytes, BlockStoreLo
     }
     // 2. transfer blocks among tiers.
     // 2.1. group blocks move plan by the destination tier.
-    Map<String, Set<BlockTransferInfo>> blocksGroupedByDestTier = new HashMap<>();
+    Map<String, Set> blocksGroupedByDestTier = new HashMap<>();
```

Comment:
Changing `Map<String, Set<BlockTransferInfo>>` to ` Map<String, Set>` will make Java compiler confused a bit in later reference of `blocksGroupedByDestTier` to justify the type of element, creating warnings.
Can you revert the change of this line?